context/app/static/js/shared-styles/inputs/MultiAutocomplete/MultiAutocomplete.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+import Autocomplete, { AutocompleteProps } from '@mui/material/Autocomplete';
+import TextField, { TextFieldProps } from '@mui/material/TextField';
+import { InputLabelProps as inputLabelProps } from '@mui/material/InputLabel';
+
+interface MultiAutocompleteTypes extends AutocompleteProps {
```

Comment:
The AutocompleteProps type is generic (its properties' types depend on the value type, whether it's a multiple autocomplete, whether it's clearable, whether it accepts freetext inputs, and optionally, the Chip component that it should use). I suggest modifying the interface:
```
interface MultiAutocompleteTypes<Value> extends AutocompleteProps<Value, true, false, false>
```
The component itself will also need a Value type parameter to support this:
```
function MultiAutocomplete<Value>({
```